colav:
  cost_function:
    P_: 1.0
    Q_: 4.0
    D_CLOSE_: 200.0
    D_SAFE_: 60.0
    K_COLL_: 0.5
    PHI_AH_: 15.0
    PHI_OT_: 68.5
    PHI_HO_: 22.5
    PHI_CR_: 68.5
    KAPPA_: 2.0
    K_P_: 4.0
    K_CHI_: 1.21
    K_DP_: 3.5
    K_DCHI_SB_: 0.9
    K_DCHI_P_: 1.2
  offsets:
    Chi_ca_: [-89.0,-75.0,-60.0,-45.0,-30.0,-15.0,0.0,15.0,30.0,45.0,60.0,75.0,89.0]
    P_ca_: [1]
  update_frequency: 1
  prediction_time: 60
  verbose: true